tests/e2e/test_vehicle.py

```diff
@@ -202,6 +202,43 @@ def test_webhooks(chevy_volt):
         assert unsubscribe._fields == ("status", "meta")
 
 
+def test_request(chevy_volt):
+    odometer = chevy_volt.request(
+        "GET", "odometer", None, {"sc-unit-system": "imperial"}
+    )
+    assert type(odometer) == types.Response
+    assert odometer.body is not None
+    assert isinstance(odometer.meta, tuple)
+    assert odometer._fields == ("body", "meta")
+    assert odometer.meta.unit_system == "imperial"
+
+
+def test_request_override_header(chevy_volt):
+    try:
+        chevy_volt.request(
+            "GET",
+            "odometer",
+            None,
+            {
+                "sc-unit-system": "imperial",
+                "Authorization": "Bearer abc",
+            },
+        )
+    except SmartcarException as sc_e:
+        assert (
+            sc_e.message
+            == "AUTHENTICATION - The authorization header is missing or malformed, or it contains invalid or expired authentication credentials. Please check for missing parameters, spelling and casing mistakes, and other syntax issues."
+        )
+
+
+def test_request_with_body(chevy_volt):
```

Comment:
any test request should work here, no reason to lock the volt during development / CI. Could update this to a brand specific endpoint in the future